### PR TITLE
List of bought rewards

### PR DESCRIPTION
**Pull request by mdobosz3**
Add serialization to the Order

### CODE REVIEW

File: app/controllers/orders_controller.rb [hunk 1]
```diff
@@ -2,12 +2,15 @@
 
 class OrdersController < ApplicationController
   before_action :authenticate_employee!
```
Comment (oskarlakner) [inline app/controllers/orders_controller.rb:4]: I'd add an empty line here

File: spec/system/order_spec.rb [hunk 2]
```diff
@@ -25,5 +27,22 @@
     within('[data-test-id="Kudo_Points"]') do
       expect(page).to have_content '0'
     end
+
+    click_link 'Orders'
+    expect(page).to have_content reward.title
+    expect(page).to have_content reward.description
+    expect(page).to have_content reward.price
+
+    sign_in admin_user
```
Comment (oskarlakner) [inline spec/system/order_spec.rb:36]: You could simplify it by using `using_session`

Screenshot 2022-03-07 at 10 14 03

**Review by oskarlakner — APPROVED** [review]

File: app/controllers/orders_controller.rb (hunk 1 above)
Comment (mdobosz3) [reply app/controllers/orders_controller.rb:4]: I changed it

File: spec/system/order_spec.rb (hunk 2 above)
Comment (mdobosz3) [reply spec/system/order_spec.rb]: I added `using_session`

**Pull request merged**